Example/shapesExample.cpp

```diff
@@ -0,0 +1,48 @@
+#include "MinGL.h"
+#include <iostream>
+
+int main()
+{
+	// Window sizes
+	const int width = 800;
+	const int height = 600;
+	MinGL minGL;
+	if (!minGL.init(width, height, "MinGL"))
+		return -1;
+
+	const char* glsl_version = "#version 130";
```

Comment:
Not sure why this glsl_version line is here!

Reply:
this line was included in [Main.cpp](https://github.com/tpk32/MinGL/blob/main/Example/Main.cpp) and i copied some code from there, that's why its here also. I don't know if it serves some purpose but the code runs fine without it.
Shall i remove this line from both [Main.cpp](https://github.com/tpk32/MinGL/blob/feature-MinGL-adding-basic-shapes/Example/Main.cpp) and [shapesExample.cpp](https://github.com/tpk32/MinGL/blob/feature-MinGL-adding-basic-shapes/Example/shapesExample.cpp)

Reply:
Hmmm. I see now. Yeah, it was probably a leftover from something I was trying to do at the time.
Can you please remove from both?

Now that I see it, I don't like that it has two main() functions. We're not JAVA! :)
I might be asking too much, but can you try moving the examples to a Header file with functions like drawCube and DrawShapes and call them from "Main.cpp"? You can leave one of the examples commented out, no problem.
Thanks a lot!